OrbitGl/CaptureSerializer.cpp

```diff
@@ -26,87 +30,107 @@
 #include "absl/strings/str_format.h"
 #include "capture_data.pb.h"
 
+static absl::flat_hash_map<const uint32_t, const std::string>
+    capture_version_to_orbit_version_map{
```

Comment:
this also works.. but now we need to keep int -> string mapping in all of our versions, where if we save it to proto - we will only need to keep the latest supported one in the code.

Reply:
One more idea: do we want to show the exact version for the user, or the range? For example, "this capture could be opened with v1.42-v1.45" (for this I should change the type of the map and the error message). In this case if capture was taken in 1.43, we won't require v1.43 to open the capture, but any version from the range. But we don't know the max version when we save the capture, and for this purpose mapping could be useful.

Reply:
I added the solution with the range of the appropriate Orbit versions that gives more information to the user.